Apps/AdminWebClient/src/ClientApp/src/components/core/CommunicationTable.vue

```diff
@@ -112,8 +116,9 @@ export default class CommunicationTable extends Vue {
         communicationTypeCode: CommunicationType.Email,
         text: "<p></p>",
         priority: 10,
-        effectiveDateTime: new Date(),
-        expiryDateTime: new Date(),
+        scheduledDateTime: moment(new Date()).toDate(),
+        effectiveDateTime: moment(new Date()).toDate(),
+        expiryDateTime: moment(new Date()).toDate(),
```

Comment:
As we default these dates (effective and expiry) to now, have you tested these changes by creating 2 Email Communications?

Reply:
Yup - no interference from my testing